.github/workflows/webcomponents.yml

```diff
@@ -0,0 +1,64 @@
+name: Artifacts
+run-name: 📦 Generate artifacts for ${{ github.event_name == 'issue_comment' && 'PR' || (github.event_name == 'release' && '🏷' || '🌱') }} ${{github.event_name == 'issue_comment' && github.event.issue.number || github.ref_name}}
+
+# This workflow runs whenever a commit is pushed on main or a release is published
+on:
+  push:
+    branches:
+      - main
+  release:
+    types: [published]
```

Comment:
is it for each release or each commit ? the PR message is confusing.

Reply:
Right, it's not mentioned in the PR but the workflow also runs when releases are created; then the `gn-wc.js` file is simply attached to it